content/test/integration/controller/active-pointers.spec.ts

```diff
@@ -178,6 +178,65 @@ loadStandaloneTestEnvironment()('Integration - Create entities', (testEnv) => {
     expect(queryResult.rows[1].pointer).toBe('1,0')
   })
 
+
+  it('when overwriting multiple scenes, unused pointers should be deleted from active-pointers table', async () => {
```

Comment:
should we test also that the field `deleter_deployment` in deployments is correctly updated? What do you think?